fe/fe-core/src/main/java/com/starrocks/catalog/lake/LakeTable.java

```diff
@@ -0,0 +1,78 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.catalog.lake;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.catalog.Column;
+import com.starrocks.catalog.DistributionInfo;
+import com.starrocks.catalog.KeysType;
+import com.starrocks.catalog.OlapTable;
+import com.starrocks.catalog.PartitionInfo;
+import com.starrocks.catalog.TableIndexes;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.common.io.Text;
+import com.starrocks.persist.gson.GsonUtils;
+import com.starrocks.server.GlobalStateMgr;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+
+/**
+ * Metadata for StarRocks lake table
+ * <p>
+ * TODO: support table api like Iceberg
+ */
+public class LakeTable extends OlapTable {
```

Comment:
Why extends OlapTable but it's not a OlapTable?

Reply:
Currently, new lake table can reuse the members of the olap table, and some new ones can be added for lake table, such as storage group. 
The most important thing is that the new lake table can use gson serialization to ensure scalability.